android/src/main/java/com/reactnativeteadssdkmodule/RNInReadAdPlacement.kt

```diff
@@ -0,0 +1,138 @@
+package com.reactnativeteadssdkmodule
+
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import com.facebook.react.bridge.*
+import tv.teads.sdk.*
+import tv.teads.sdk.renderer.InReadAdView
+import java.util.*
+
+
+class RNInReadAdPlacement(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
```

Comment:
I asked it because when a consumer integrates our sdk [we provide a callback for this functions interfaces](https://github.com/teads/TeadsSDK-android/blob/3f3a5a3a7a44b5dc3fef1929eaad8afdd88924af/TeadsSDKDemo/app/src/main/java/tv/teads/teadssdkdemo/format/inread/InReadScrollViewFragment.kt#L49), then the consumer knows when the functions were called

Reply:
It's not a test class i could be the final integration, I tried to do it as it is in Flutter plugin for the callback.
Here is the class : [FLTinReadAdPlacement](https://github.com/teads/TeadsSDK-flutter/blob/main/packages/TeadsSDK/android/src/main/kotlin/tv/teads/teadssdkflutter/teads_sdk_flutter/FLTInReadAdPlacement.kt)
The publisher will not be able to add something to these functions (for now).
I've tried to look at how to do it, but it seems complicated.
Let me know if we need to treat it another way than how it is in Flutter.

Reply:
Ok so this should be solved later?

Reply:
It could be, I don't now what are the goals for it 
Since we don't have it on Flutter I thought we will do the same here, but we could solve it post-production too.
Do you think it's a requirement? or it can wait like Flutter?